css/navbar.css

```diff
@@ -3,6 +3,10 @@
 :root {
   --color-nav: #1d1283;
   --color-nav-text: #ffffff;
+  --dropdown-background: #ffffff;
+  --dropdown-border: #000000;
+  --dropdown-text: #000000;
```

Comment:
Do not repeat the HEX, as then there is no point in making variables.
Do rename the existing variables as per the color they have.
For example:
```CSS
--color-white: #ffffff;
--color-black: #000000;
```

Reply:
ok